lncore/peers.go

```diff
@@ -1,9 +1,46 @@
 package lncore
 
+import (
+	"fmt"
+
+	"github.com/mit-dci/lit/bech32"
+)
+
 // LnAddr is just a bech32-encoded pubkey.
 // TODO Move this to another package so it's more obviously not *just* IO-related.
 type LnAddr string
```

Comment:
This type definition should really be thought about. My suggestion is that we keep this as string and remove the typecast for now until we find purpose for it. Unless you have other plans :)

Reply:
The point of it being a separate type is so that you have the guarantee that it's a properly-formatted ln address.  It also changes the way you think about processing addresses so that it's though of more like how you'd parse binary data into a structure.

Of course in Go this is a weak guarantee, but if it was tracked as a `string` then it could lead to problems when refactoring (and accidentally removing checks) that lead to random errors.  That's why I removed all the other force casts to/from the `LnAddr` type.  With this you have some typing guarantee that you won't (can't) forget to verify anything.